README.md

```diff
@@ -155,6 +155,8 @@ merged into the RFC repository as a markdown file. At that point the RFC is
   - In most cases, the FCP period is quiet, and the RFC is either merged or
     closed. However, sometimes substantial new arguments or ideas are raised,
     the FCP is canceled, and the RFC goes back into development mode.
+  - It is a this stage that a number is assigned to the RFC file and it is the
+    number of the PR.
```

Comment:
yeah we usually assign the number (rename the file) when we merge. Although it doesn't really matter if someone does it before, that'd be ok too.

Reply:
Can we maybe move this up to the bullet where it says "create a file named `0000-foo`". It could say "create a file named `0000-my-feature.md`. Don't assign the RFC a number yet -- if the RFC is accepted, we'll rename the file when we merge the PR."

Reply:
Sure thing, Niko. I'll fix this tomorrow morning.